Rakefile

```diff
@@ -44,6 +46,13 @@ namespace :travis do
   end
 end
 
+desc "Parallel spec tests"
+task :parallel_spec do
+  require 'parallel_tests/cli'
+  Rake::Task[:spec_prep].invoke
+  ParallelTests::CLI.new.run('--type test -t rspec spec/classes spec/defines spec/unit'.split)
```

Comment:
Damn just realised I forgot functions, I did think the speed increase was a little _too_ impressive...